BMM.UI.Android/Application/Constants/AndroidScreenSizesHeight.cs

```diff
@@ -0,0 +1,8 @@
+namespace BMM.UI.Droid.Application.Constants
+{
+    public class AndroidScreenSizesHeight
+    {
+        public const int HD = 1280;
+        public const int FullHD = 1280;
```

Comment:
that seems wrong? Shouldn't FullHD be 1920?

Reply:
You're right, good catch.